examples/mappings/requirements.txt

```diff
@@ -0,0 +1,6 @@
+oteapi-core>=0.6.1,<1
+#oteapi-dlite>0.1.6,<1  # update once after migrating to pydantic v2
+oteapi-dlite @ git+https://github.com/EMMC-ASBL/oteapi-dlite.git@cwa/close-187-migrate-to-pydantic-v2
+DLite-Python>=0.5.1,<1
```

Comment:
```suggestion
```

Reply:
I think it is better to list DLite-Python explicitly rather than having it install implicitly by oteapi-dlite.